src/cosalib/kubevirt.py

```diff
@@ -25,19 +18,26 @@ def __init__(self, **kwargs):
         self.mutate_callback = self.write_oci
         self.mutate_callback_creates_final_image = True
 
-    def write_oci(self, image_name):
+    def write_oci(self, _):
         """
         Take the qcow2 base image and convert it to an oci-archive.
         """
-        buildah_base_argv = buildah_base_args()
-        final_img = os.path.join(os.path.abspath(self.build_dir),
-                                 self.image_name)
-        cmd = buildah_base_argv + ["from", "scratch"]
-        buildah_img = runcmd(cmd, capture_output=True).stdout.decode("utf-8").strip()
-        runcmd(buildah_base_argv + ["add", "--chmod", "0555", buildah_img, image_name, "/disk/coreos.img"])
-        cmd = buildah_base_argv + ["commit", buildah_img]
-        digest = runcmd(cmd, capture_output=True).stdout.decode("utf-8").strip()
-        runcmd(buildah_base_argv + ["push", "--format", "oci", digest, f"oci-archive:{final_img}"])
+        ctxdir = self._tmpdir
+        final_img = os.path.join(os.path.abspath(self.build_dir), self.image_name)
+        # Create the Containerfile to use for the build
+        with open(os.path.join(ctxdir, "Containerfile"), "w") as f:
+            f.write(f"FROM scratch\nADD {self.image_name_base}.{self.image_format} /disk/\n")
```

Comment:
Before, we named the disk to `coreos.img` in the image. Do we not need to retain that?